tests/test_utils.py

```diff
@@ -161,3 +165,105 @@ def test_fs_str():
 def test_fs_str_with_bytes():
     with raises(AssertionError):
         fs_str(b"whatever")
+
+
+@mark.parametrize(
+    "cli_args, expected_command",
+    [
+        # Check empty args
+        ([], "pip-compile"),
+        # Check all options which will be excluded from command
+        (["-v"], "pip-compile"),
+        (["--verbose"], "pip-compile"),
+        (["-n"], "pip-compile"),
+        (["--dry-run"], "pip-compile"),
+        (["-q"], "pip-compile"),
+        (["--quiet"], "pip-compile"),
+        (["-r"], "pip-compile"),
+        (["--rebuild"], "pip-compile"),
+        (["-U"], "pip-compile"),
+        (["--upgrade"], "pip-compile"),
+        (["-P", "django"], "pip-compile"),
+        (["--upgrade-package", "django"], "pip-compile"),
+        # Check options
+        (["--max-rounds", "42"], "pip-compile --max-rounds=42"),
+        (["--index-url", "https://foo"], "pip-compile --index-url=https://foo"),
+        # Check that short options will be expanded to long options
+        (["-p"], "pip-compile --pre"),
+        (["-f", "links"], "pip-compile --find-links=links"),
+        (["-i", "https://foo"], "pip-compile --index-url=https://foo"),
+        # Check positive flags
+        (["--generate-hashes"], "pip-compile --generate-hashes"),
+        (["--pre"], "pip-compile --pre"),
+        (["--allow-unsafe"], "pip-compile --allow-unsafe"),
+        # Check negative flags
+        (["--no-index"], "pip-compile --no-index"),
+        (["--no-emit-trusted-host"], "pip-compile --no-emit-trusted-host"),
+        (["--no-annotate"], "pip-compile --no-annotate"),
+        # Check that default values will be removed from the command
+        (["--emit-trusted-host"], "pip-compile"),
+        (["--annotate"], "pip-compile"),
+        (["--index"], "pip-compile"),
+        (["--max-rounds=10"], "pip-compile"),
+        (["--no-build-isolation"], "pip-compile"),
```

Comment:
can you add a test for:
`(["--output-file", "requirements.txt", "requirements.in"], "pip-compile")`

Reply:
Okay, but why it should be `pip-compile` in the expected command?

Reply:
Do you think `pip-compile --output-file requirements.txt requirements.in` should be normalized to `pip-compile`?

Reply:
Tested in `test_get_compile_command_with_files`.